<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingInterface</class>
 <widget class="QWidget" name="SettingInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="first_up">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="first_down">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="first_left">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="first_right">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="second_up">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="second_down">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="second_left">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="second_right">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="third_up">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="third_down">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="third_left">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="third_right">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forth_up">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forth_down">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>120</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forth_left">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>120</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forth_right">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>110</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="first_attack">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="first_special">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="second_attack">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="second_special">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="third_attack">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="third_special">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forth_attack">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forth_special">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
